dist/source/repo-creator/sponsorship.html

```diff
@@ -0,0 +1,40 @@
+<template>
+	<require from="./choose-repository.css"></require>
+
+	<div class="container-fluid choose-repository">
+		<div class="row">
+			<div class="col-xs-12 panel-group" id="accordion">
+				<div class="panel panel-default">
+					<div class="panel-heading panel-top">
+						<h4 class="panel-title">
+							<a class="accordion-toggle" data-toggle="collapse" data-parent="#accordion" click.delegate="showSponsored()">
+							MY SPONSORSHIPS<i class="indicator glyphicon pull-right" class.bind="isSponsoredSelected ? 'glyphicon-chevron-down' : 'glyphicon-chevron-up'"></i>
+							</a>
+						</h4>
+					</div>
+					<div class="panel-collapse collapse" class.bind="isSponsoredSelected ? 'in' : ''">
```

Comment:
Most of the HTML can go away since this page isn't actually using an accordion anymore, such as this collapsible panel.